### DISCUSSION TITLE
Code for the Cardivascular model

### DISCUSSION THREAD

**Discussion by J-Anoop** [ThummeTo/FMIFlux.jl#99] [General]
Hello Tobias, 

Your article titled "Hybrid modeling of the human cardiovascular system using NeuralFMUs", mentions about you sharing the source code here with the FMIFlux.jl repo. I am not able to locate it, could you please point me to it?

**Comment by ThummeTo** [comment]
Dear @J-Anoop,
you are totally right about that. Since the paper release in 2021 there where multiple major changes in FMIFlux.jl, so unfortunately the original code isn't working anymore (out of the box). A big ToDo is to re-run the experiments from the paper and re-construct this as example code again. One of our students is already on this. By the way I am sure that we will outperform the original results by far, because there where several improvements (technical, as well as in methodology) since then. But we still need some additional time for that. Until, you may want to check out the [tutorial from the JuliaCon2023](https://github.com/ThummeTo/FMIFlux.jl/blob/main/examples/src/juliacon_2023.ipynb) that handles the general procedure of setup/training a NeuralFMU.
Best regards!

**Reply by J-Anoop** [reply]
Ok Sir, Thank you for the update! 
I did watch your Juliacon 2023 workshop video, and then looked into your articles etc. That's some impressive work you guys are doing, resolving a pain point with N-ODE's application in real world cases.

Actively following your work.
Regards,
Anoop.
